modules/tide_ckeditor/js/ckeditor5_plugins/callout/src/calloutcommand.js

```diff
@@ -0,0 +1,17 @@
+import { Plugin } from 'ckeditor5/src/core';
+
+export default class CalloutCommand extends Plugin {
+
+    execute(){
+        const model = this.editor.model;
+        const document = model.document;
+        model.change(writer => {
+            const blocks = Array.from(document.selection.getSelectedBlocks())
+            for ( const block of blocks ) {
+                if ( !block.is( 'element', 'tidecallout' ) ) {
+                    writer.rename( block, 'tidecallout' );
+                }
+            }
+        });
+    }
+}
```

Comment:
Just a thought - some of these js files are 2 space indent and some are 4. Do we have a preferred style for JS files in a Drupal module? Is there a Drupal standard we should adopt here?

ping @anthony-malkoun

Reply:
Hmm, I thought there was some `.editorconfig` for this. Worth looking into.

Reply:
hey @krakerag , @anthony-malkoun . just reformat the code to follow Drupal js style.

Reply:
I think that works. I might do some looking up as to what the Drupal standard is for in-project JS